lib/index.js

```diff
@@ -86,26 +93,35 @@ module.exports = {
   convert (language, variant, request, options, callback) {
     var convert, main;
 
-    if (!sdk.Request.isRequest(request)) {
-      return callback('Codegen~convert: Invalid request');
-    }
+    try {
+      if (!sdk.Request.isRequest(request)) {
+        throw TypeError('Codegen~convert: Invalid request');
```

Comment:
If the provided request is not of the right sdk.Request type. Then there is no point in throwing error, you can simply callback with the error.